.github/workflows/pages.yml

```diff
@@ -0,0 +1,47 @@
+name: docs
+on:
```

Comment:
I am not sure because it is new for me, but I'd try with:
on:
  release:
    types: [published]

Reply:
I can change it when we do a release. I know it works but still want to test it on push

Reply:
Makes sense